arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -180,3 +183,25 @@ extension UIColor {
     // The translucent background color for status labels.
     class var statusLabelBackgroundColor: UIColor { return UIColor(named: "statusLabelBackgroundColor")! }
 }
+
+extension AppDelegate {
+    /// License the app with ArcGIS Runtime deployment license keys.
+    ///
+    /// - Note: An invalid key does not throw an exception, but simply fails to license the app,
+    ///   falling back to Developer Mode (which will display a watermark on the map view).
+    static func licenseApplication() {
```

Comment:
I would suggest making this an _instance_ method called `license()` on `UIApplication`. Since `AppDelegate.application(_:didFinishLaunchingWithOptions:)` is passed the application, it can call the method on the passed in application:

```swift
application.license()
```

I think that reads nicer than:

```swift
AppDelegate.licenseApplication()
```